packages/eslint-plugin/src/rules/no-draft-imports.ts

```diff
@@ -0,0 +1,63 @@
+import { Rule } from "eslint"
+
+module.exports = {
+  meta: {
+    type: "problem",
+    docs: {
+      description: "Reports usages of draft components",
+      category: "Possible Errors",
+      recommended: true,
+    },
+    fixable: "code",
+    schema: [],
+  },
+  create(context: Rule.RuleContext): Rule.NodeListener {
+    return {
+      VariableDeclarator(node) {
+        if (!node.init || node.init.type !== "CallExpression") return
+        if (node.init.arguments.length > 0) {
+          node.init.arguments.forEach(arg => {
+            if (arg.type !== "Literal" || typeof arg.value !== "string") return
+            if (arg.value.includes("@kaizen/draft")) {
+              context.report({
+                message: `Unexpected draft import ${arg.value}`,
+                node,
+                fix(fixer) {
+                  if (typeof arg.value === "string" && arg.raw) {
+                    const newValue = arg?.raw.replace(
+                      "kaizen/draft-",
+                      "kaizen/"
+                    )
+                    return fixer.replaceText(arg, newValue)
+                  }
+                  return null
+                },
+              })
+            }
+          })
+        }
```

Comment:
Im curious, should we add a check here to see if the `callee` is `require`? Or, `require.resolve`, `require.resolve.paths` etc.  Otherwise (if i'm understanding correctly) will this potentially pickup any function that has `@kaizen/draft` anywhere in its parameters?

I wouldn't say this is a PR blocking problem as we can always amend later if what i say has value.

Reply:
For extra dilligence we could also add a check to see if it's `import(...)`.

It's not like anyone at CA is doing this though so up to you.

Reply:
Oh! I did not even think of this use case - thank you. I'll pop in a test for it while I'm at it.